cloud/services/compute/loadbalancers/reconcile.go

```diff
@@ -58,6 +61,17 @@ func (s *Service) Reconcile(ctx context.Context) error {
 	}
 
 	lbSpec := s.scope.LoadBalancer()
+	createBootstrapInstanceGroup := false
+	if lbSpec.InternalLoadBalancer != nil {
```

Comment:
Do we need the if statement here? With the default value set to false on deref it may be redundant.

Reply:
Good point. Removed.